docs/help-topics/stonesoup-whatsnext/stonesoup-whatsnext.yml

```diff
@@ -38,12 +38,11 @@
 
 - name: stonesoup-whatsnext-install-github-app
   tags:
-  title: Install the Stonesoup GitHub app 
+  title: Install our GitHub app
   content: |-
     To customize your build pipelines, install the GitHub app. After you install the GitHub app, it creates webhooks for your repositories and running builds. The GitHub app also tests your pull requests for you to view before merging your pull requests. 
 
-    1. Go to the GitHub repository: https://github.com/apps/appstudio-staging-ci/.  
-    links: []
```

Comment:
I think there was a bug in quickstart that required this array to exist, would you mind checking it this is still the case with the latest quickstart version?